cms/templates/home_page.html

```diff
@@ -70,42 +54,6 @@ <h3 class="featured-product-header-title col-8">{{ product_cards_section_title }
       </div>
     </div>
   </div>
-{% else %}
-  <div class="course-section">
-    <div class="container">
-      {% if product_cards_section_title %}
-        <h1>{{ product_cards_section_title }}</h1>
-      {% endif %}
-      <ul class="course-listing card-listing">
-        {% for product in products %}
-          {% if not product.is_program %}
-            <li>
-          <div class="course-card highlight-card clickable">
-            <a href="{{ product.url_path }}" class="course-card-link">
-              <img src="{% feature_img_src product.feature_image %}" alt="">
-              <div class="course-info">
-                <h2 class="course-card-title">
-                    {{ product.title }}
-                </h2>
-                <div class="course-card-description">
-                  {{ product.description | richtext }}
-                </div>
-                {% if product.start_date %}
-                  <span class="date">
-                    Start date: {{ product.start_date | date:"F j, Y" }}
-                  </span>
-                {% endif %}
-              </div>
-            </a>
-          </div>
-        </li>
-          {% endif %}
-        {% endfor %}
-      </ul>
-    </div>
-  </div>
-{% endif %}
-{% if show_home_page_video_component and page.video_component_title %}
```

Comment:
The component does not show. If I remember correctly, it was decided way long ago that we should show nothing if there was no title since it would not look... correct. We can circle back to it if you think it's an issue, but the page still loads and everything - was just a way to stop the page from looking odd with it not there as well as to handle if  they wanted to remove the component.